src/core/public/doc_links/doc_links_service.ts

```diff
@@ -6,7 +6,7 @@
  * Side Public License, v 1.
  */
 
-import { deepFreeze } from '@kbn/std';
+import { getDocLinks, getDocLinksMeta, DocLinks } from '@kbn/doc-links';
```

Comment:
nit: split type imports from runtime
```suggestion
import type { DocLinks } from '@kbn/doc-links';
import { getDocLinks, getDocLinksMeta } from '@kbn/doc-links';
```